### ISSUE TITLE
groupyr and numpy

### ISSUE THREAD

**Issue by sung** [gregbellan/Stabl#1]
I followed your instruction manual, but it seems the required version of `groupyr` (0.3.2) conflicts with the version of `numpy` you demanded, e.g.
```shell
pip install numpy==1.23.2
...
ERROR: pip's dependency resolver does not currently take into account all the packages that are installed. This behaviour is the source of the following dependency conflicts.
groupyr 0.3.2 requires numpy==1.21.5, but you have numpy 1.23.2 which is incompatible.
```
Any thought?

**Comment by gbellan** [comment]
That's normal ! the library should work anyway ;)

**Comment by sung** [comment]
Then, why not remove the dependency of using `numpy` 1.23.2 thought the document and the repo?

> That's normal ! the library should work anyway ;)

**Comment by xavdurand** [comment]
Unfortunately, some of the features that we are using are not in the numpy version `numpy==1.21.5`, but in older version.

The `groupyr` package used to implement `Sparse Group Lasso` and `Sparse Group Logit Lasso` fixed its version of numpy to `numpy==1.21.5` even if it is working well with older version. So, we just updated the `numpy` version to `numpy==1.23.2` because it was our setup.

> Note: There could be an issue with numba if the numpy version is too old, depending on the version of numba.

**Comment by xavdurand** [comment]
We have published a light-weight version of Stabl with the release `v1.0.0-lw`. Please refer to this version if you have issues with Julia and Groupyr installation.
I also close this issue, as it is resolved by the light-weight release ✨
